renovate.json

```diff
@@ -24,13 +24,11 @@
     },
     {
       "groupName": "MUI public packages",
```

Comment:
This doesn't apply just to MUI public packages anymore

Reply:
Do we still need this group if it is coming from `mui-public`? 🤔

Reply:
This just configures it with `"allowedVersions": "!/-dev/"`. not sure whether that is still relevant

Reply:
There are still some packages published from the core repo. However I would discourage using the canary publish script. I think by now all latest versions are higher than the dev versions